packages/cfd/src/Containers/cfd-dashboard.tsx

```diff
@@ -731,7 +729,6 @@ export default withRouter(
         dxtrade_disabled_signup_types: client.dxtrade_disabled_signup_types,
         has_maltainvest_account: client.has_maltainvest_account,
         can_upgrade_to: client.can_upgrade_to,
-        account_settings: client.account_settings,
```

Comment:
`account_settings` are redundant here as they are only used to get `residence` from it. It's already done above with `country: client.account_settings.residence,` so there's no need for `account_settings`, `country` can be used directly.
Doing it in this PR because it's affecting component props in the test file.